components/shared-styles/settings.scss

```diff
@@ -1,6 +1,67 @@
-@import "colors.scss";
+/** Colors*/
+// blues
+$xlightblue: #C3DDF9;
+$hoverBlue: #32538D;
+$byuNavy: #002e5d;
+$byuRoyal: #0057b8;
```

Comment:
This isn't BYU Royal. It should be #0062B8. A lot of these colors don't match the spec on brand site (https://brand.byu.edu/web-toolkit/color/). Also, do we need all these colors?

Reply:
Good catch - I just grabbed the CSS from the BYU-web-docs repo Nate made, but we should at least have the correct BYU Royal, navy, etc. I'll work on trimming down the colors this afternoon and let you know once I've committed a fix.

Reply:
Ok, I've fixed the royal and removed the unused SCSS variables. The menu still needs work on mobile but that can wait for a separate PR unless you haven't gotten too far on your branch.

Reply:
I haven't gotten far. Go ahead and do what you'd like.